templates/style/_navbar.scss

```diff
@@ -122,8 +128,21 @@ div.nav-container {
            See https://github.com/rust-lang/docs.rs/issues/1669.
            */
        .pure-menu-item a {
-            /* 0.5 em is the padding */
-            max-height: calc(#{$top-navbar-height} - 0.5em * 2);
+            max-height: 100%;
+            overflow-x: hidden;
```

Comment:
I think these lines are unnecessary. When I remove these properties locally, the unwanted grey rectangles (which are tiny vertical scrollbars) go away. Also, with these properties removed, even if the menu items' font size is weird, it doesn't interfere with the main page.

Reply:
In fact, I think this whole CSS rule (including the comment about what happens with a locally installed old Fira Sans) should get deleted. Switching to the flex layout seems like it sufficed to fix the issue with overflowing menu titles, even without this rule.

Reply:
(I just tested in a locally installed Firefox, with the Fira Sans from https://github.com/rust-lang/docs.rs/issues/1669#issuecomment-1068396583, at these commits:

1. ad8d4fe33ba6d48669175221f3b651eb5e1e78ec (i.e. right before #1711)
2. 0ae58075652454f6bd535e39fdcd7be27f846fd5 (this branch)
3. 0ae58075652454f6bd535e39fdcd7be27f846fd5 with this CSS rule removed

And I can confirm that at (1) I have the overflow issue. Both (2) and (3) fix the overflow issue, and (3) additionally fixes the scrollbar issue introduced in this branch.

BTW the scrollbar issue seems to be Chrome-only.

Reply:
Without this rule:

![Screenshot from 2022-04-13 16-51-54](https://user-images.githubusercontent.com/3050060/163208725-612f9f47-5ede-4ef4-9a22-a24186109d8a.png)

With this rule:

![Screenshot from 2022-04-13 16-52-21](https://user-images.githubusercontent.com/3050060/163208717-5bacff3d-3832-490a-be89-d282af392133.png)

So we definitely need it.